src/app/pages/session/edit/[id]/index.vue

```diff
@@ -183,6 +183,7 @@
             :title="t('logout')"
             @click="signOut"
           >
+            <template #iconSecondary> </template>
```

Comment:
Does the following work as well? Because ` ` may be added as a literal space character in the HTML which is generally to avoid because it can lead to weird styling issues sometimes.
```suggestion
            <template #iconSecondary />
```

Reply:
It does work but your suggestion seems better.